src/ChannelServer/World/Dungeons/DungeonManager.cs

```diff
@@ -121,7 +121,7 @@ private Dungeon CreateDungeon(string dungeonName, int itemId, Creature creature)
 			{
 				// Create new dungeon if there is non yet or it's Tuesday
 				var existing = this.Get(a => a.Name == dungeonName && a.ItemId == itemId);
-				if (existing == null || ErinnTime.Now.Month == 2 || ChannelServer.Instance.Conf.World.PrivateDungeons)
```

Comment:
Even if it's incorrect in its current form, doesn't it make more sense to have it than to remove it o.o? People expect a private dungeon on Tuesday.